crates/ruff_cli/src/printer.rs

```diff
@@ -439,3 +482,29 @@ fn print_fix_summary(writer: &mut dyn Write, fixed: &FxHashMap<String, FixTable>
     }
     Ok(())
 }
+
+/// Build the displayed fix status message depending on the types of the remaining fixes.
+fn display_violations(safe_remaining: i32, unsafe_remaining: i32) -> String {
+    let prefix = format!("[{}]", "*".cyan());
+    let mut fix_status = prefix;
+
+    if safe_remaining > 0 {
+        fix_status =
+            format!("{fix_status} {safe_remaining} potentially fixable with the --fix option.");
+    }
+    if unsafe_remaining > 0 {
+        let (line_break, extra_prefix) = if safe_remaining > 0 {
+            // Avoid re-allocating prefix.
+            ("\n", format!("[{}]", "*".cyan()))
+        } else {
+            ("", String::new())
+        };
+
+        let total = safe_remaining + unsafe_remaining;
+        fix_status = format!(
+            "{fix_status}{line_break}{extra_prefix} {total} potentially fixable with the --fix --unsafe options."
+        );
+    }
```

Comment:
```suggestion
    if safe_remaining > 0 {
        format!("{prefix} {safe_remaining} potentially fixable with the --fix option.")
    }
    if unsafe_remaining > 0 {
        let (line_break, extra_prefix) = if safe_remaining > 0 {
            // Avoid re-allocating prefix.
            ("\n", format!("[{}]", "*".cyan()))
        } else {
            ("", String::new())
        };

        let total = safe_remaining + unsafe_remaining;
        format!(
            "{prefix}{line_break}{extra_prefix} {total} potentially fixable with the --fix --unsafe options."
        )
    }
```

Reply:
I don't think this will work - we have to allocate `fix_status` so that we can carry through the first option if there are fixes fixable by both `--fix` and `--fix-unsafe`